spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -1183,6 +1184,9 @@ public InterpreterResult interpret(String line, InterpreterContext context) {
   public InterpreterResult interpret(String[] lines, InterpreterContext context) {
     synchronized (this) {
       z.setGui(context.getGui());
+      String paraText = context.getParagraphText();
+      sc.setCallSite(new CallSite(Utils.getJobShortText(paraText),
```

Comment:
@zjffdu I didnot find api to `setLocalProperty()` in R.    
 Any idea/pointer on how to implement his with sparkR?